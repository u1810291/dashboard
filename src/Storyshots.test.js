import initStoryshots from '@storybook/addon-storyshots'
// import { imageSnapshot } from '@storybook/addon-storyshots-puppeteer'

jest.mock('../.storybook/intlDecorator', () => fn => fn())
jest.mock('../.storybook/storyStylesDecorator', () => fn => fn())

initStoryshots({ suite: 'Fragments storyshots', storyKindRegex: /^fragments/ })
initStoryshots({
  suite: 'Components storyshots',
  storyKindRegex: /^components/
  // test: imageSnapshot()
})
