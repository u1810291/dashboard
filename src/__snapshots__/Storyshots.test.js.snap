// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components storyshots components/Card Default 1`] = `
<div
  className="background-white border-transparent  css-16liexg-Card"
>
  Panel content
</div>
`;

exports[`Components storyshots components/Card Left aside element 1`] = `
<div
  className="background-active border-transparent  css-16liexg-Card"
>
  <h3>
    Some title
    <p>
      Panel content
    </p>
  </h3>
</div>
`;

exports[`Components storyshots components/Click Default 1`] = `
<div
  className="background-white border-transparent css-1opud1h-Click"
  onClick={[Function]}
  onKeyPress={[Function]}
  onMouseOut={[Function]}
  onMouseOver={[Function]}
  role="button"
  tabIndex={0}
>
  La Button
</div>
`;

exports[`Components storyshots components/Click Link 1`] = `
<a
  className="background-active border-transparent css-1opud1h-Click"
  href="/"
  onClick={[Function]}
  onKeyPress={[Function]}
  onMouseOut={[Function]}
  onMouseOver={[Function]}
  role="button"
  tabIndex={0}
>
  La Button
</a>
`;

exports[`Components storyshots components/Click Not inline 1`] = `
<div
  className="background-white border-transparent css-1isq01e-Click"
  onClick={[Function]}
  onKeyPress={[Function]}
  onMouseOut={[Function]}
  onMouseOver={[Function]}
  role="button"
  tabIndex={0}
>
  La Button
</div>
`;

exports[`Components storyshots components/Click With Shadow 1`] = `
<div
  className="background-white border-transparent css-105t1i5-Click"
  onClick={[Function]}
  onKeyPress={[Function]}
  onMouseOut={[Function]}
  onMouseOver={[Function]}
  role="button"
  tabIndex={0}
>
  <strong>
    La Button
  </strong>
</div>
`;

exports[`Components storyshots components/Details Default 1`] = `
<div
  className="css-1qo54az"
>
  <h3
    className=""
    onClick={[Function]}
  >
    Is Mati secure?
  </h3>
</div>
`;

exports[`Components storyshots components/Details Pre-opened 1`] = `
<div
  className="css-1qo54az"
>
  <h3
    className="opened"
    onClick={[Function]}
  >
    Is Mati secure?
  </h3>
  <div
    className="text-secondary"
  >
    <p>
      In short, yes it is. In fact, the lack of security in most of the applications we use today is the reason we built Mati.
    </p>
    <p>
      We strongly believe the only path forward in securing our digital lives is to lift this responsibility away from users and put it on the technology around us. As we see it, security is either missing altogether or too frustrating to use.
    </p>
    <p>
      That is why we tried to minimize the difficulty and make Mati easy to use, but keep security as the highest priority.We use highly secure cryptographic methods to secure your data at Mati. All of your sensitive data is encrypted using standards-based crypto-algorithms.
    </p>
    <p>
      Unlike proprietary algorithms, standards-based algorithms have gone through public scrutiny by industry and security experts that reduces the chance of any inherent weaknesses or vulnerabilities.We also continuously make the best effort to secure our servers online and client applications locally.
    </p>
  </div>
</div>
`;

exports[`Components storyshots components/Items Default 1`] = `
<div
  className="css-1274dzz"
>
  <div
    style={
      Object {
        "background": "orange",
        "minHeight": "80px",
        "minWidth": "100px",
      }
    }
  />
  <div
    style={
      Object {
        "background": "blue",
        "minHeight": "160px",
        "minWidth": "40px",
      }
    }
  />
  <div
    style={
      Object {
        "background": "orange",
        "minHeight": "80px",
        "minWidth": "100px",
      }
    }
  />
  <div
    style={
      Object {
        "background": "blue",
        "minHeight": "160px",
        "minWidth": "40px",
      }
    }
  />
</div>
`;

exports[`Components storyshots components/YoutubeOverlay Default 1`] = `
Array [
  " ",
  <button
    onClick={[Function]}
  >
    Sample Overlay
  </button>,
]
`;

exports[`Fragments storyshots fragments/info/Faq Default 1`] = `
<div
  className="background-white border-transparent  css-16liexg-Card"
>
  <h2>
    fragments.info.faq.title
  </h2>
  <div
    className="css-uqbefl"
  >
    <div
      className="css-1qo54az"
    >
      <h3
        className=""
        onClick={[Function]}
      >
        How will Mati verify my users?
      </h3>
    </div>
    <hr />
    <div
      className="css-1qo54az"
    >
      <h3
        className=""
        onClick={[Function]}
      >
        Is Mati secure?
      </h3>
    </div>
    <hr />
    <div
      className="css-1qo54az"
    >
      <h3
        className=""
        onClick={[Function]}
      >
        Which countries do you support?
      </h3>
    </div>
  </div>
</div>
`;

exports[`Fragments storyshots fragments/info/Features Default 1`] = `
<div
  className="css-uqbefl"
>
  <h2>
    Security Features
  </h2>
  <div
    className="css-1tphajl"
  >
    <div
      className="background-white border-transparent css-1t7c5od-Click"
      onClick={[Function]}
      onKeyPress={[Function]}
      onMouseOut={[Function]}
      onMouseOver={[Function]}
      role="button"
      tabIndex={0}
    >
      <img
        alt="liveness"
        src={
          Object {
            "ReactComponent": [Function],
            "__esModule": true,
            "default": "liveness.svg",
          }
        }
      />
      <h2
        className="text-light"
      >
        fragments.info.features.liveness.title
      </h2>
    </div>
    <div
      className="background-white border-transparent css-1t7c5od-Click"
      onClick={[Function]}
      onKeyPress={[Function]}
      onMouseOut={[Function]}
      onMouseOver={[Function]}
      role="button"
      tabIndex={0}
    >
      <img
        alt="template-matching"
        src={
          Object {
            "ReactComponent": [Function],
            "__esModule": true,
            "default": "template-matching.svg",
          }
        }
      />
      <h2
        className="text-light"
      >
        fragments.info.features.template-matching.title
      </h2>
    </div>
    <div
      className="background-white border-transparent css-1t7c5od-Click"
      onClick={[Function]}
      onKeyPress={[Function]}
      onMouseOut={[Function]}
      onMouseOver={[Function]}
      role="button"
      tabIndex={0}
    >
      <img
        alt="watchlist"
        src={
          Object {
            "ReactComponent": [Function],
            "__esModule": true,
            "default": "watchlist.svg",
          }
        }
      />
      <h2
        className="text-light"
      >
        fragments.info.features.watchlist.title
      </h2>
    </div>
    <div
      className="background-white border-transparent css-1t7c5od-Click"
      onClick={[Function]}
      onKeyPress={[Function]}
      onMouseOut={[Function]}
      onMouseOver={[Function]}
      role="button"
      tabIndex={0}
    >
      <img
        alt="facematch"
        src={
          Object {
            "ReactComponent": [Function],
            "__esModule": true,
            "default": "facematch.svg",
          }
        }
      />
      <h2
        className="text-light"
      >
        fragments.info.features.facematch.title
      </h2>
    </div>
    <div
      className="background-white border-transparent css-1t7c5od-Click"
      onClick={[Function]}
      onKeyPress={[Function]}
      onMouseOut={[Function]}
      onMouseOver={[Function]}
      role="button"
      tabIndex={0}
    >
      <img
        alt="document-reading"
        src={
          Object {
            "ReactComponent": [Function],
            "__esModule": true,
            "default": "document-reading.svg",
          }
        }
      />
      <h2
        className="text-light"
      >
        fragments.info.features.document-reading.title
      </h2>
    </div>
    <div
      className="background-white border-transparent css-1t7c5od-Click"
      onClick={[Function]}
      onKeyPress={[Function]}
      onMouseOut={[Function]}
      onMouseOver={[Function]}
      role="button"
      tabIndex={0}
    >
      <img
        alt="curp-validation"
        src={
          Object {
            "ReactComponent": [Function],
            "__esModule": true,
            "default": "curp-validation.svg",
          }
        }
      />
      <h2
        className="text-light"
      >
        fragments.info.features.curp-validation.title
      </h2>
    </div>
    <div
      className="background-white border-transparent css-1t7c5od-Click"
      onClick={[Function]}
      onKeyPress={[Function]}
      onMouseOut={[Function]}
      onMouseOver={[Function]}
      role="button"
      tabIndex={0}
    >
      <img
        alt="alteration-detection"
        src={
          Object {
            "ReactComponent": [Function],
            "__esModule": true,
            "default": "alteration-detection.svg",
          }
        }
      />
      <h2
        className="text-light"
      >
        fragments.info.features.alteration-detection.title
      </h2>
    </div>
  </div>
</div>
`;

exports[`Fragments storyshots fragments/integration/GooglePlayLink Default 1`] = `
<div
  className="background-white border-transparent  css-16liexg-Card"
>
  <div
    className="css-uqbefl"
  >
    <div
      className="css-6aswc1"
    >
      <h3>
        developers.demo.demo
        <p>
          developers.demo.description
        </p>
      </h3>
      <div
        className="css-5w61r0"
      >
        <a
          href="https://play.google.com/store/apps/details?id=com.matilock.mati_global_demo"
          rel="noopener noreferrer"
          target="_blank"
        >
          <img
            alt="Google Play"
            src="google-play.png"
          />
        </a>
        <p>
          developers.demo.buttonDescription
        </p>
      </div>
      <div
        className="css-1qo54az"
      >
        <h3>
          developers.demo.clientId
          <p>
            developers.demo.clientIdDescription
          </p>
        </h3>
        <div
          className="background-lightergray border-transparent  css-or8dev-Card"
        >
          <strong>
            <code>
              CLIENT ID NUMBER
            </code>
          </strong>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Fragments storyshots fragments/integration/InstallationGuidePanels ApplicationsPanel 1`] = `
<div
  className="background-white border-transparent  css-1vw7yee-Card"
>
  <aside
    className="aside background-apricot"
  >
    <h2
      className="text-darkapricot"
    >
      2
    </h2>
  </aside>
  <div
    className="background-white border-transparent  css-oiazn5-Card"
  >
    <div
      className="css-6aswc1"
    >
      <div
        className="css-1qo54az"
      >
        <h2>
          fragments.integration.installation-guide-panels.applications-panel.title
        </h2>
        <p>
          fragments.integration.installation-guide-panels.applications-panel.message
        </p>
      </div>
      <a
        aria-current={null}
        className="default primary"
        href="/integration/applications"
        onClick={[Function]}
        type="button"
      >
        fragments.integration.installation-guide-panels.applications-panel.button
      </a>
    </div>
    <svg>
      applications.svg
    </svg>
  </div>
</div>
`;

exports[`Fragments storyshots fragments/integration/InstallationGuidePanels InstallFrontendCodePanel 1`] = `
<div
  className="background-white border-transparent  css-1vw7yee-Card"
>
  <aside
    className="aside background-apricot"
  >
    <h2
      className="text-darkapricot"
    >
      3
    </h2>
  </aside>
  <div
    className="background-white border-transparent  css-oiazn5-Card"
  >
    <div
      className="css-6aswc1"
    >
      <div
        className="css-1qo54az"
      >
        <h2>
          fragments.integration.installation-guide-panels.jssdk-panel.title
        </h2>
        <p>
          fragments.integration.installation-guide-panels.jssdk-panel.message
        </p>
      </div>
      <a
        aria-current={null}
        className="default primary"
        href="/integration/integration-code"
        onClick={[Function]}
        type="button"
      >
        fragments.integration.installation-guide-panels.jssdk-panel.button
      </a>
    </div>
    <svg
      className="cursor-pointer"
      onClick={[Function]}
    >
      video.svg
    </svg>
  </div>
</div>
`;

exports[`Fragments storyshots fragments/integration/InstallationGuidePanels LetsStartPanel 1`] = `
<div
  className="background-white border-transparent  css-1vw7yee-Card"
>
  <aside
    className="aside background-apricot"
  >
    <h2
      className="text-darkapricot"
    />
  </aside>
  <div
    className="background-apricot border-transparent  css-oiazn5-Card"
  >
    <div
      className="css-uqbefl"
    >
      <h2>
        fragments.integration.installation-guide-panels.start-panel.title
      </h2>
      <p>
        fragments.integration.installation-guide-panels.start-panel.message
      </p>
    </div>
    <svg>
      lets-start.svg
    </svg>
  </div>
</div>
`;

exports[`Fragments storyshots fragments/integration/InstallationGuidePanels SetWebhooksPanel 1`] = `
<div
  className="background-white border-transparent  css-1vw7yee-Card"
>
  <aside
    className="aside background-apricot"
  >
    <h2
      className="text-darkapricot"
    >
      4
    </h2>
  </aside>
  <div
    className="background-white border-transparent  css-oiazn5-Card"
  >
    <div
      className="css-6aswc1"
    >
      <div
        className="css-1qo54az"
      >
        <h2>
          fragments.integration.installation-guide-panels.webhooks-panel.title
        </h2>
        <p>
          fragments.integration.installation-guide-panels.webhooks-panel.message
        </p>
      </div>
      <a
        aria-current={null}
        className="default primary"
        href="/integration/webhooks"
        onClick={[Function]}
        type="button"
      >
        fragments.integration.installation-guide-panels.webhooks-panel.button
      </a>
    </div>
    <div
      className="css-uqbefl"
    >
      <svg
        className="cursor-pointer"
        onClick={[Function]}
      >
        video.svg
      </svg>
      <section>
        fragments.integration.installation-guide-panels.webhooks-panel.link
        : 
        <a
          href="https://docs.getmati.com"
        >
          docs.getmati.com
        </a>
      </section>
    </div>
  </div>
</div>
`;

exports[`Fragments storyshots fragments/integration/InstallationGuidePanels VerificationPanel 1`] = `
<div
  className="background-white border-transparent  css-1vw7yee-Card"
>
  <aside
    className="aside background-apricot"
  >
    <h2
      className="text-darkapricot"
    >
      1
    </h2>
  </aside>
  <div
    className="background-white border-transparent  css-oiazn5-Card"
  >
    <div
      className="css-6aswc1"
    >
      <div
        className="css-1qo54az"
      >
        <h2>
          fragments.integration.installation-guide-panels.verification-panel.title
        </h2>
        <p>
          fragments.integration.installation-guide-panels.verification-panel.message
        </p>
      </div>
      <a
        aria-current="page"
        className="default primary active"
        href="/"
        onClick={[Function]}
        style={Object {}}
        type="button"
      >
        fragments.integration.installation-guide-panels.verification-panel.button
      </a>
    </div>
    <svg>
      verification.svg
    </svg>
  </div>
</div>
`;

exports[`Fragments storyshots fragments/integration/WebhooksDocsPanel Default 1`] = `
<div
  className="background-white border-transparent  css-dubmwf-Card"
>
  <div
    className="css-6aswc1"
  >
    <h2>
      fragments.integration.webhooks-docs-panel.title
      <p>
        fragments.integration.webhooks-docs-panel.subtitle
      </p>
    </h2>
    <div
      className="css-b54nyf"
    >
      <svg>
        picture.svg
      </svg>
      <svg
        className="cursor-pointer"
        onClick={[Function]}
      >
        installation-video.svg
      </svg>
    </div>
    <section>
      <a
        className="background-active border-transparent css-1opud1h-Click"
        href="https://docs.getmati.com"
        onClick={[Function]}
        onKeyPress={[Function]}
        onMouseOut={[Function]}
        onMouseOver={[Function]}
        rel="noopener noreferrer"
        role="button"
        tabIndex={0}
        target="_blank"
      >
        fragments.integration.webhooks-docs-panel.button
      </a>
    </section>
  </div>
</div>
`;

exports[`Fragments storyshots fragments/integration/WebhooksTestPanel default 1`] = `
<div
  className="background-softpink border-transparent  css-dubmwf-Card"
>
  <div
    className="css-uqbefl"
  >
    <h2>
      fragments.integration.webhooks-test-panel.title
    </h2>
    <div
      className="css-1uehtmq"
    >
      <ol
        className="mgi-list"
      >
        <li>
          fragments.integration.webhooks-test-panel.list.0
        </li>
        <li>
          fragments.integration.webhooks-test-panel.list.1
        </li>
        <li>
          fragments.integration.webhooks-test-panel.list.2
        </li>
        <li>
          fragments.integration.webhooks-test-panel.list.3
        </li>
      </ol>
      <svg>
        picture.svg
      </svg>
    </div>
    <div
      className="css-bukpbg"
    >
      <h3
        className="text-darkpink"
      >
        <small>
          fragments.integration.webhooks-test-panel.pro-tip.caption
        </small>
      </h3>
      <div
        className="css-dso5ro"
      >
        <div
          className="background-transparent border-darkpink  css-or8dev-Card"
        >
          <small
            className="text-darkpink"
          >
            fragments.integration.webhooks-test-panel.pro-tip.message
          </small>
        </div>
      </div>
    </div>
  </div>
</div>
`;
